core/lexicon/en/file.inc.php

```diff
@@ -76,6 +76,9 @@
 $_lang['file_name'] = 'File Name';
 $_lang['file_quick_create'] = 'Quick Create File';
 $_lang['file_quick_update'] = 'Quick Edit File';
+$_lang['file_package_err_invalid_directory'] = 'Packages directory doesnt appear to exist!';
+$_lang['file_package_err_invalid_name'] = 'This file [[+name]] has an invalid name for the transport package';
+$_lang['file_package_err_invalid_type'] = 'This file does not appear to be a transport package';
```

Comment:
`doesnt` or `does not`?

Reply:
English isn't everyone's first language.

Reply:
But consistency is better interface language than English.